webrtc-stats.html

```diff
@@ -1269,6 +1271,44 @@
                 </p>
               </dd>
               <dt>
+                <dfn><code>playoutTimestamp</code></dfn> of type <span class="idlMemberType"><a>
+                DOMHighResTimeStamp</a></span>
+              </dt>
+              <dd>
+                <p>
+                  Only valid for remote sources. This is the estimated playout time of this track.
+                  The playout time is the NTP timestamp of the last played out audio sample or video
+                  frame that has a known timestamp (from an RTCP SR packet mapping RTP timestamps to
+                  NTP timestamps), extrapolated with the time elapsed since it was played out. This
+                  is the "current time" of the track in NTP clock time of the sender and can be
+                  present even if there is no audio or video currently playing.
+                </p>
+                  This can be useful for estimating how much audio and video is out of sync for two
+                  tracks from the same source, <code>audioTrackStats.playoutTimestamp -
+                  videoTrackStats.playoutTimestamp</code>.
+                <p>
+                </p>
+              </dd>
+              <dt>
+                <dfn><code>captureTimestamp</code></dfn> of type <span class="idlMemberType"><a>
+                DOMHighResTimeStamp</a></span>
+              </dt>
+              <dd>
+                <p>
+                  Only valid for remote sources. This is the estimated capture time of this track,
+                  i.e. what the "current time" is for the remote capturer. It is estimated by
+                  looking at the latest RTCP SR packet containing an NTP timestamp. The difference
+                  between the local clock when the packet arrived and the NTP timestamp gives an
+                  estimate of half the round trip time. The <code>captureTimestamp</code> is the
+                  current local time plus the latest half-RTT estimate and can be present even if
```

Comment:
I had this question before, there are no synchronized clocks, so the one-way delay cannot be calculated by taking a difference between NTP timestamps in the SR and the local time.